compiler+runtime/include/cpp/jank/codegen/llvm_processor.hpp

```diff
@@ -66,6 +66,17 @@ namespace jank::codegen
     function,
     eval
   };
+  /* There are three places where a var-root could be generated depending on different circumstances.
```

Comment:
This comment is not connected to the `compilation_target` enum. Please use a new line for spacing.